python/modules/__init__.py

```diff
@@ -11,3 +12,22 @@ def build_problem_report_for_connections(family, problem_code, problem_str):
             "problem-code": problem_code,
             "explain": problem_str
         })
+
+    def validate_common_message_blocks(self, msg, family):
+        """
+        Validate threading, timing blocks in messages and return error message to sender when invalid
+        """
+        try:
+            msg.validate_common_blocks()
+            return 1
+        except ValidationException as e:
+            their_did = msg.context.get('from_did')
+            if their_did:
+                err_msg = self.build_problem_report_for_connections(family,
+                                                                    e.error_code, str(e.exception))
+                await self.agent.send_message_to_agent(their_did, err_msg)
```

Comment:
```
indy-agent/python/modules/__init__.py", line 28
    await self.agent.send_message_to_agent(their_did, err_msg)
    ^
SyntaxError: 'await' outside async function
```

Reply:
Fixed this. But my tests fail now. Let me check. I did some final reorg after my tests were working and didn't check after the reorg